buildchow/templates/core-jobs.yaml

```diff
@@ -42,17 +44,17 @@
       - 'devstudio.versionwatch_{devstudio-stream}'
       - 'jbosstools-build-sites.aggregate.site_{stream}'
       - 'jbosstools-buildflow_{stream}'
- 
-- project:
+
+ - project:
     name: jbosstools-mars
-    disabled: false
+    disabled: true
     stream:
       4.3.mars
     devstudio-stream:
       9.0.mars
-    target-platform-version: 4.50.0.Beta1
-    target-platform-version-maximum: 4.50.0.Beta1
-    branch: jbosstools-4.3.0.Beta1x
+    target-platform-version: 4.50.0.CR1
+    target-platform-version-maximum: 4.50.0.CR1
+    branch: jbosstools-4.3.x
```

Comment:
I mean I would like to run these builds before actually making them live on the production jenkins instance. Wether that is beta2x or something else is secondary - but sure, beta2x works for me.